### PR TITLE
Adding documentation for the ActiveSpeakerAPI

### PR DESCRIPTION
**Pull request by kohlikohl**
The change involves:

- removal of redundant code from existing samples
- addition of documentation of the ActiveSpeaker API
- addition of warning notes to P2P samples that they are not supported in Google Chrome

### CODE REVIEW

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md) | :warning:Warning | | [Details](#user-content-Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md)

<a name="Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md"></a>
### [Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md)
 - **Line unspecified**: **[Warning]** Failed to find Git account for the committer maknoll@microsoft.com. Solution: 1) File creator to associate email address maknoll@microsoft.com with his or her Git account, or 2) Anyone to explicitly add &quot;author: &lt;Git account&gt;&quot; into markdown YAML header.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C13%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201701130134489119-161%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/TOC.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/TOC.md) | :warning:Warning | | [Details](#user-content-Skype/TOC.md)
[Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md) | :warning:Warning | | [Details](#user-content-Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md)

<a name="Skype/TOC.md"></a>
### [Skype/TOC.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/TOC.md)
 - **Line unspecified**: **[Warning]** Invalid file link:(~/PTVideoGroupGoogleChrome.md).

<a name="Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md"></a>
### [Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md)
 - **Line unspecified**: **[Warning]** Failed to find Git account for the committer maknoll@microsoft.com. Solution: 1) File creator to associate email address maknoll@microsoft.com with his or her Git account, or 2) Anyone to explicitly add &quot;author: &lt;Git account&gt;&quot; into markdown YAML header.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C13%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201701130143195976-161%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C13%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201701130146072828-161%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 1]
```diff
@@ -6,83 +6,113 @@
...
-
-We can add participants to the conversation by calling add(...) providing a SIP URI on the participants collection of the conversation object.  We can use the videoService on the conversation object and call start() to initate the call.
-
-After the conversation and video modality are established we can begin communicating with the remote parties.  When finished click the end button to terminate the conversation.
+1. As a first step we need to create a conversation
 
+    ```js
+        var conversation = application.conversationsManager.createConversation();
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: delete this tab - `var` should be aligned with the ```

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 2]
```diff
@@ -6,83 +6,113 @@
...
-
-When either the selfParticipant or other persons are conencted to video we also need to setup the DOM element where the video should be displayed.  This configuration can be handled by getting access to the sink object by walking from the person object to the video modality to the channels collection choosing the first, or channels(0), which gives us access to the stream object which has a source object which finally points us to the sink object.  The sink object has a format property that can accept video formatting options such as Stretch, Fit, and Crop.  The sink object also exposes a container property where we can provide a DOM element where the video will be inserted.
-
-We can add participants to the conversation by calling add(...) providing a SIP URI on the participants collection of the conversation object.  We can use the videoService on the conversation object and call start() to initate the call.
-
-After the conversation and video modality are established we can begin communicating with the remote parties.  When finished click the end button to terminate the conversation.
+1. As a first step we need to create a conversation
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: In general I prefer imperative instructions in these docs; eg. "First, create a conversation" and "Add a listener to display self video..." rather than "we need to...", "we are adding..." Not sure if MS has a preferred convention - I guess @JohnAustin-MSFT  or @srividhc would be the people to ask

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 3]
```diff
@@ -6,83 +6,113 @@
...
+    ```
 
-### Start a group video conversation
+1. Now we are adding a listener to display our own video when the camera is turned on.
+When the listener is triggered, we can assign a DOM node as the container for the video.
 
-1. Start group video conversation, and track participant events 
+    ```js
+        conversation.selfParticipant.video.state.when('Connected', function () {
+            // video is availabe ... lets assign a container.
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: put this comment on a second line - looks pretty weird in MSDN when it just breaks mid-word

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 4]
```diff
@@ -6,83 +6,113 @@
...
+            conversation.selfParticipant.video.channels(0).stream.source.sink.format('Stretch'); // formats include: 'Stretch', 'Fit' and 'Crop'
+            conversation.selfParticipant.video.channels(0).stream.source.sink.container(/* DOM node such as a DIV */);
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: This line also breaks weirdly in MSDN

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 5]
```diff
@@ -6,83 +6,113 @@
...
+            // the video will be rendered automatically
+        });
+    ```
+1. To render the video of the other participants in the conversation we add another listener that notifies us when
+another participant joins the conversation.
+
+    > [!Warning]
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: I'd make this "IMPORTANT," - I think we should save WARNINGS for things like making sure not to post decrypted HTTPS traffic that might contain passwords on stackoverflow, mostly because the "warning" is pretty aggressive-looking in MSDN and we don't want to overuse it

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 6]
```diff
@@ -6,83 +6,113 @@
...
+    > Group video for **Google Chrome** is rendered using a different API.
+    > Please see [Group in Google Chrome](PTVideoGroupGoogleChrome.md) for more details.
 
-  ```js
-    var conversationsManager = application.conversationsManager;
-    var id = content.querySelector('.id').value;
-    var id2 = content.querySelector('.id2').value;
-    conversation = conversationsManager.createConversation();
+    ```js
+        conversation.participants.added(function (person) {
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: same comment about align code lines with the ```

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 7]
```diff
@@ -6,83 +6,113 @@
...
-                // set up remote video container
-                if (Object.keys(videoMap).length === 1) {
-                    videoMap[person.displayName()] = 2;
-                    setupContainer(person, 'large', content.querySelector('.remoteVideoContainer2'));
-                }
-                else {
-                    videoMap[person.displayName()] = 1;
-                    setupContainer(person, 'large', content.querySelector('.remoteVideoContainer1'));
-                }
+                // now we can render it
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: Not your change, but I think this is wrong. We need to add another listener for isVideoOn, and set isStarted(true/false) when isVideoOn() becomes true/false respectively

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 8]
```diff
@@ -6,83 +6,113 @@
...
 
     ```js
-    listeners.push(person.video.channels(0).isVideoOn.when(true, function () {
-        // person.displayName() started streaming their video
-    }));
-    listeners.push(person.video.channels(0).isVideoOn.when(false, function () {
-       // person.displayName() stopped streaming their video
-    }));
-  ```
+        conversation.participants.add('<participant1.id>');
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: same indentation comment

Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: also maybe stick with your "sip:XXX" syntax rather than changing it up

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 9]
```diff
@@ -6,83 +6,113 @@
...
+            // handle error
+        });
+    ```
 
-3. End the conversation
+## Ending the conversation
 
-  ```js
+ ```js
     conversation.leave().then(function () {
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: indentation

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 10]
```diff
@@ -6,83 +6,113 @@
...
         // handle error
     }).then(function () {
         // clean up operations
     });
-  ```
+```
+
+## Complete code sample
+
+Below is the code alltogether
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: *altogether. But I'd just say something like, "The following is a complete code sample to start a new group video call."

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 11]
```diff
@@ -6,83 +6,113 @@
...
+        conversation.participants.add('<participant2.id>');
+
+        // starting the video conference
+        conversation.videoService.start().then(null, function (error) {
+            // handle error
+        });
+    ```
 
-3. End the conversation
+## Ending the conversation
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: I'd also include the videoService disconnected listener here that you removed from the above snippet

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 12]
```diff
@@ -6,83 +6,113 @@
...
+## Complete code sample
+
+Below is the code alltogether
+
+```js
+var conversation = application.conversationsManager.createConversation();
+
+conversation.selfParticipant.video.state.when('Connected', function () {
+    // video is availabe ... lets assign a container.
+    conversation.selfParticipant.video.channels(0).stream.source.sink.format('Stretch'); // formats include: 'Stretch', 'Fit' and 'Crop'
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: break into 2 lines

File: Skype/WebSDK/docs/PTVideoGroup.md [hunk 13]
```diff
@@ -6,83 +6,113 @@
...
+            // now we can render it
+            person.video.channels(0).isStarted(true);
+
+            // NOTE: .isStarted() only needs to be called for remote participants in group conversations
+            // it dictates wether or not the participant's video should be rendered
+        });
+    });
+});
+
+conversation.participants.add('<participant1.id>');
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroup.md]: same comment about 'sip:xxx'

File: Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md [hunk 14]
```diff
@@ -0,0 +1,38 @@
+
+# Audio/Video in Google Chrome
+
+
+
+ _**Applies to:** Skype for Business 2015_
+
+At the moment we only support one incoming video channel in Google Chrome.
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md]: Maybe rephrase this, since this makes it sound like our fault and not a technological limitation imposed on us by WebRTC. It also makes it sound like at some point we're going to add this 'support,' which as far as I know isn't going to happen

File: Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+ _**Applies to:** Skype for Business 2015_
+
+At the moment we only support one incoming video channel in Google Chrome. 
+To still render video of multiple participants in a conversation we have introduced the **Active Speaker** API.
+
+Through the API the video of the currently speaking participant is sent over the one available channel.
+
+## Active Speaker API
+
+### How fo I know when to use the Active Speaker API?
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md]: *do

File: Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+```js
+    var conversation = application.conversationsManager.createConversation();
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md]: indentation

File: Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md (hunk 15 above)
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md]: Maybe include a little text here explaining, "If you want your app to support group video on either Chrome or Microsoft Edge or IE, you will need to add code to detect what APIs to use when rendering a group av call."

File: Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+```js
+    // set up listener to detect if someone is actively speaking
+    conversation.videoService.activeSpeaker.participant.changed(function (participant) {
+        // participant.displayName() is speaking
+
+        // lets render the video if the currently speaking participant is not myself
+        if (participant !== conversation.selfParticipant) {
+            var channel = conversation.videoService.activeSpeaker.channel;
+            channel.source.sink.container(/* DOM element such as DIV*/);
+            channel.isStarted(true);
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md]: Still need to handle isVideoOn as in group video - otherwise weird things happen when participants start and stop video I think

File: Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md [hunk 18]
```diff
@@ -0,0 +1,38 @@
...
+    var conversation = application.conversationsManager.createConversation();
+    if (conversation.videoService.videoMode() === 'ActiveSpeaker') {
+        // use the Active Speaker API
+    }
+```
+
+### How do I render the Active Speaker Channel?
+```js
+    // set up listener to detect if someone is actively speaking
+    conversation.videoService.activeSpeaker.participant.changed(function (participant) {
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md]: Should make it clear that this listener should be added after video is known to be connected; ie. selfparticipant.video.state() == connected

File: Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md [hunk 19]
```diff
@@ -0,0 +1,38 @@
...
+### How fo I know when to use the Active Speaker API?
+```js
+    var conversation = application.conversationsManager.createConversation();
+    if (conversation.videoService.videoMode() === 'ActiveSpeaker') {
+        // use the Active Speaker API
+    }
+```
+
+### How do I render the Active Speaker Channel?
+```js
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md]: Indentation

File: Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md [hunk 20]
```diff
@@ -0,0 +1,38 @@
...
+        // lets render the video if the currently speaking participant is not myself
+        if (participant !== conversation.selfParticipant) {
+            var channel = conversation.videoService.activeSpeaker.channel;
+            channel.source.sink.container(/* DOM element such as DIV*/);
+            channel.isStarted(true);
+        }
+    });
+```
+
+When the active speaker changes, the video will update to display the stream of the currently speaking person.
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md]: Maybe add "You don't need to call any APIs to change whose video is shown through the active speaker channel - this is automatically determined by the media stack."

File: Skype/WebSDK/docs/PTVideoIncomingP2P.md [hunk 21]
```diff
@@ -4,6 +4,9 @@
 
  _**Applies to:** Skype for Business 2015_
 
+> [!IMPORTANT]
+> Meetings with one remote participant (P2P/1:1) are not supported in **Google Chrome** at this point.
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoIncomingP2P.md]: see meetings comment

File: Skype/WebSDK/docs/PTVideoIncomingP2P.md [hunk 22]
```diff
@@ -20,39 +23,37 @@ After the conversation and video modality are established we can begin communica
...
-    listeners.push(conversationsManager.conversations.added(function (conv) {
-        conversation = conv;
-        function setupContainer(person, size, videoDiv) {
-            person.video.channels(0).stream.source.sink.format('Stretch');
-            person.video.channels(0).stream.source.sink.container(videoDiv);
-        }
-        listeners.push(conversation.videoService.accept.enabled.when(true, function () {
-            // showModal('Accept incoming video invitation?');
-            if (USER_ACCEPT_INCOMING_VIDEO) {
+    conversationsManager.conversations.added(function (conversation) {
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoIncomingP2P.md]: indentation

File: Skype/WebSDK/docs/PTVideoIncomingP2P.md [hunk 23]
```diff
@@ -20,39 +23,37 @@ After the conversation and video modality are established we can begin communica
...
-        }));
+        });
 
-        listeners.push(conversation.selfParticipant.video.state.when('Connected', function () {
+        conversation.selfParticipant.video.state.when('Connected', function () {
             // set up remote video container
-            setupContainer(conversation.selfParticipant, 'small', content.querySelector('.selfVideoContainer'));
-        }));
+            conversation.selfParticipant.video.channels(0).stream.source.sink.format('Stretch');
+            conversation.selfParticipant.video.channels(0).stream.source.sink.container(/* DOM node such as DIV */);
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoIncomingP2P.md]: this line breaks weirdly

File: Skype/WebSDK/docs/PTVideoOutgoingP2P.md [hunk 24]
```diff
@@ -4,6 +4,9 @@
 
  _**Applies to:** Skype for Business 2015_
 
+> [!IMPORTANT]
+> Meetings with one remote participant (P2P/1:1) are not supported in **Google Chrome** at this point.
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoOutgoingP2P.md]: meetings note

File: Skype/WebSDK/docs/PTVideoOutgoingP2P.md [hunk 25]
```diff
@@ -23,32 +26,29 @@ After the conversation and video modality are established we can begin communica
...
+    conversation = conversationsManager.getConversation('sip:xxx');
 
-    listeners.push(conversation.selfParticipant.video.state.when('Connected', function () {
+    conversation.selfParticipant.video.state.when('Connected', function () {
         // set up self video container
-        setupContainer(conversation.selfParticipant, 'small', content.querySelector('.selfVideoContainer'));
-    }));
-    listeners.push(conversation.participants.added(function (person) {
+        conversation.selfParticipant.video.channels(0).stream.source.sink.format('Stretch');
+        conversation.selfParticipant.video.channels(0).stream.source.sink.container(/* DOM node such as DIV */);
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoOutgoingP2P.md]: breaks weirdly

File: Skype/WebSDK/docs/PTVideoOutgoingP2P.md [hunk 26]
```diff
@@ -23,32 +26,29 @@ After the conversation and video modality are established we can begin communica
...
+    });
+    conversation.participants.added(function (person) {
         console.log(person.displayName() + ' has joined the conversation');
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoOutgoingP2P.md]: usually this is just a comment (// person.displayName() + ' has joined the conversation') - we should be consistent

File: Skype/WebSDK/docs/PTVideoP2PEscalation.md [hunk 27]
```diff
@@ -4,6 +4,9 @@
 
  _**Applies to:** Skype for Business 2015_
 
+> [!IMPORTANT]
+> Meetings with one remote participant (P2P/1:1) are not supported in **Google Chrome** at this point.
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoP2PEscalation.md]: meetings note

File: Skype/WebSDK/docs/PTVideoP2PEscalation.md [hunk 28]
```diff
@@ -20,44 +23,33 @@ After the conversation and video modality are established we can begin communica
...
-        person.video.channels(0).stream.source.sink.container(videoDiv);
-    }
+    conversation = conversationsManager.getConversation('sip:xxx');
 
-    listeners.push(conversation.selfParticipant.video.state.when('Connected', function () {
+    conversation.selfParticipant.video.state.when('Connected', function () {
         // set up local video container
-        setupContainer(conversation.selfParticipant, 'large', content.querySelector('.selfVideoContainer'));
+        conversation.selfParticipant.video.channels(0).stream.source.sink.format('Stretch');
+        conversation.selfParticipant.video.channels(0).stream.source.sink.container(/* DOM node such as DIV */);
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoP2PEscalation.md]: breaks weirdly

File: Skype/WebSDK/docs/PTVideoGroup.md (hunk 3 above)
Comment (kohlikohl) [inline Skype/WebSDK/docs/PTVideoGroup.md]: This is an issue of the code formatter which breaks words up into multiple lines depending on the viewport width.

**Comment by ajkher** [comment]
Are we adding a sample as well leveraging the ActiveSpeaker API? @kohlikohl

File: Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md (hunk 18 above)
Comment (kohlikohl) [inline Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md]: To my knowledge this listener can be added even before. It doesn't matter if the participant is UNDEFINED to begin with.

**Comment by kohlikohl** [comment]
@ajkher Yes! Seth has worked on a sample which can be found here https://github.com/OfficeDev/skype-docs/tree/sethli/update-auth-meeting-join-sample

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/WebSDK/docs/includes/P2PChromeWarning.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/WebSDK/docs/includes/P2PChromeWarning.md) | :warning:Warning | | [Details](#user-content-Skype/WebSDK/docs/includes/P2PChromeWarning.md)

<a name="Skype/WebSDK/docs/includes/P2PChromeWarning.md"></a>
### [Skype/WebSDK/docs/includes/P2PChromeWarning.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/WebSDK/docs/includes/P2PChromeWarning.md)
 - **Line unspecified**: **[Warning]** title is mandatory for each page, please make sure your content file meet at least one of below requirements: &quot;#&quot; exist in MD for &lt;h1&gt; or &quot;title&quot; meta exist in MD YAML block

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C13%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201701132307104536-161%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/WebSDK/docs/includes/P2PChromeWarning.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/WebSDK/docs/includes/P2PChromeWarning.md) | :warning:Warning | | [Details](#user-content-Skype/WebSDK/docs/includes/P2PChromeWarning.md)

<a name="Skype/WebSDK/docs/includes/P2PChromeWarning.md"></a>
### [Skype/WebSDK/docs/includes/P2PChromeWarning.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/WebSDK/docs/includes/P2PChromeWarning.md)
 - **Line unspecified**: **[Warning]** title is mandatory for each page, please make sure your content file meet at least one of below requirements: &quot;#&quot; exist in MD for &lt;h1&gt; or &quot;title&quot; meta exist in MD YAML block

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C13%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201701132310317596-161%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md [hunk 29]
```diff
@@ -0,0 +1,54 @@
...
+The Skype Web SDK differentiates rendering of group videos for different browsers:
+* Google Chrome
+* and all other browsers
+
+## Google Chrome
+
+In Google Chrome, due to technical limitations, the video streams of the remote participants are transported over one channel. This 
+channel is accessible through the **Active Speaker API**.
+
+The channel always containes the video of the currently speaking participant. This updates automatically
```
Comment (slifland37) [inline Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md]: *contains

File: Skype/WebSDK/docs/includes/P2PChromeWarning.md [hunk 30]
```diff
@@ -0,0 +1,2 @@
+> [!IMPORTANT]
+> P2P calls are not supported in **Google Chrome**.
```
Comment (slifland37) [inline Skype/WebSDK/docs/includes/P2PChromeWarning.md]: Maybe P2P AV calls?

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/WebSDK/docs/includes/P2PChromeWarning.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/WebSDK/docs/includes/P2PChromeWarning.md) | :warning:Warning | | [Details](#user-content-Skype/WebSDK/docs/includes/P2PChromeWarning.md)

<a name="Skype/WebSDK/docs/includes/P2PChromeWarning.md"></a>
### [Skype/WebSDK/docs/includes/P2PChromeWarning.md](https://github.com/OfficeDev/skype-docs/blob/maknoll/activeSpeaker/Skype/WebSDK/docs/includes/P2PChromeWarning.md)
 - **Line unspecified**: **[Warning]** title is mandatory for each page, please make sure your content file meet at least one of below requirements: &quot;#&quot; exist in MD for &lt;h1&gt; or &quot;title&quot; meta exist in MD YAML block

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C18%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201701182322437772-161%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kohlikohl** [comment]
@JohnAustin-MSFT I liked the Active Speaker API text reference to the API reference.
Please have another look.

If all ok. I'll go ahead and merge to master.
